src/imitation/reg/__init__.py

```diff
@@ -0,0 +1,86 @@
+"""Implements a variety of reguarlization techniques for NN weights."""
+
+import abc
+from typing import Dict
+
+import torch as th
+from torch import optim
+
+
+class Regularizer:
```

Comment:
this should inherit from `abc.ABC` given it has abstract methods

Reply:
Thanks for spotting this! Didn't remember seeing this in the linter